master/master.cfg

```diff
@@ -252,6 +253,9 @@ for branch_num, (git_url, branchname, git_branch) in enumerate(git_branches):
             # disable GitHub notifications for unstable builders
             if stability == STABLE:
                 github_status_builders.append(buildername)
+                # Only Tier-1 and Tier-2 builders can block a release
+                if tier in (TIER_1, TIER_2):
```

Comment:
builtins.NameError: name 'TIER_1' is not defined

Reply:
Fixed ;-)